client_device/python/config.yaml

```diff
@@ -0,0 +1,8 @@
+measurements:
+  cpu_total_load:
+    enabled: true
+  ram_used:
+    enabled: true
+  network_load:
+    enabled: true
+    link_address: 192.168.0.105
```

Comment:
you should comment this line to define  what line_address means? Maybe the uploaded version should not contain your IP address (ok, it is local address, but still)